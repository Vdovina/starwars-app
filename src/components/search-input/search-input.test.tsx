import { create, act as testAct, ReactTestRenderer } from 'react-test-renderer';
import { SearchInput } from './';

describe('SearchInput Tests', () => {
  it('render', () => {
    let component: ReactTestRenderer;
    testAct(() => {
      component = create(<SearchInput value="" onChange={() => {}} />);
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
