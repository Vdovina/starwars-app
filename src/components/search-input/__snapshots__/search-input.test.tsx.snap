// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput Tests render 1`] = `
<div
  className="input__wrapper"
>
  <div
    className="input__container"
  >
    <div
      className="input__first-icon"
    >
      <svg
        fill="none"
        height="15"
        viewBox="0 0 15 15"
        width="15"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M14.4062 13.2109L10.2864 9.09043C10.9503 8.17805 11.3075 7.07846 11.3064 5.95008C11.3064 2.99666 8.9035 0.59375 5.95008 0.59375C2.99666 0.59375 0.59375 2.99666 0.59375 5.95008C0.59375 8.9035 2.99666 11.3064 5.95008 11.3064C7.07846 11.3075 8.17805 10.9503 9.09043 10.2864L13.2109 14.4062L14.4062 13.2109ZM5.95008 9.61471C5.22519 9.61477 4.51656 9.39987 3.91381 8.99719C3.31106 8.5945 2.84126 8.02212 2.56383 7.35242C2.28639 6.68272 2.21379 5.94579 2.35519 5.23483C2.49659 4.52387 2.84565 3.8708 3.35823 3.35823C3.8708 2.84565 4.52387 2.49659 5.23483 2.35519C5.94579 2.21379 6.68272 2.28639 7.35242 2.56383C8.02212 2.84126 8.5945 3.31106 8.99719 3.91381C9.39987 4.51656 9.61477 5.22519 9.61471 5.95008C9.61356 6.92165 9.2271 7.8531 8.5401 8.5401C7.8531 9.2271 6.92165 9.61356 5.95008 9.61471V9.61471Z"
          fill="currentColor"
        />
      </svg>
    </div>
    <input
      className="input__component input__with-icon"
      onChange={[Function]}
      placeholder="Search characters"
      value=""
    />
    <div
      className="input__actions"
    />
  </div>
</div>
`;
